README.md

```diff
@@ -61,3 +61,21 @@ Convert a JSON representation back into an immutable object
 ### `transit.withFilter(function) => transit`
 
 Create a modified version of the transit API that deeply applies the provided filter function to all immutable collections before serialising. Can be used to exclude entries.
+
+### `transit.withRecords([recordClasses], optionalFilterFunction) => transit`
```

Comment:
Yeah that's th API that makes more sense to me as well, I just left it this way because it's how @thomasboyt implemented it. I can change it though.